packages/webapp/src/utils/utils.tsx

```diff
@@ -218,10 +218,20 @@ export function formatFrequency(frequency: string): string {
         hours: 'h',
         hour: 'h',
         days: 'd',
-        day: 'd'
+        day: 'd',
+        months: 'mon',
+        month: 'mon',
```

Comment:
why not `M` less clear but consistent in term of size?

Reply:
I thought about that. As you said it is consistent but less clear. I don't mind either way. @khaliqgant any preference?

Reply:
not pushing just saying 👍🏻

Reply:
month abbreviation should actually be `mo` if singular and `mos` if plural